src/main/java/services/ImageTransferService.java

```diff
@@ -104,8 +104,12 @@
  * ----
  *
  * At that stage, if the transfer's phase is xref:types-image_transfer_phase[paused_system], then the session was
- * not successfully established. One possible reason for that is that the ovirt-imageio-daemon is not running
+ * not successfully established. One possible reason for that is that the ovirt-imageio is not running
```

Comment:
I would also remove the "the" before ovirt-imageio-daemon. It was probably added because it's normal to refer to a daemon with "the", like any other concrete noun. It looks weird with ovirt-imageio because that is probably a service, which doesn't normally have the definite article because it is abstract. 

If you can fix the grammar and spelling of these sentences, that would be a plus.